modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -222,48 +222,50 @@ Any optional fields can be left empty and a default will be selected.
 ? Domain prefix: <domain_prefix> <1>
 ? Deploy cluster with Hosted Control Plane (optional): No
 ? Create cluster admin user: Yes <2>
-? Username: user-admin <2>
-? Password: [? for help] *************** <2>
-? OpenShift version: 4.17.0 <3>
-? Configure the use of IMDSv2 for ec2 instances optional/required (optional): <4>
-I: Using arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Installer-Role for the Installer role <5>
+? Create custom password for cluster admin: No <3>
+I: cluster admin user is cluster-admin
+I: cluster admin password is abcd1-eFgh2-jKLm3-NopQ4
```

Comment:
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-reference

I had the security team contact me in the past about using password syntax that represents real passwords.

![Screenshot from 2024-10-02 12-05-26](https://github.com/user-attachments/assets/b1b98a20-aef9-46b3-be53-4ef05c62184a)

Reply:
Roger that, thankyou!